test/integration/stamp_editor_spec.js

```diff
@@ -0,0 +1,129 @@
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          if (browserName === "firefox") {
+            pending(
+              "Disabled in Firefox, because of https://bugzilla.mozilla.org/1553847."
+            );
+          }
+
+          await page.click("#editorStamp");
+
+          const rect = await page.$eval(".annotationEditorLayer", el => {
+            // With Chrome something is wrong when serializing a DomRect,
+            // hence we extract the values and just return them.
+            const { x, y } = el.getBoundingClientRect();
+            return { x, y };
+          });
+
+          await page.mouse.click(rect.x + 100, rect.y + 100);
+          const input = await page.$("#stampEditorFileInput");
+          await input.uploadFile(
+            `${path.join(__dirname, "../images/firefox_logo.png")}`
+          );
+
+          await page.waitForTimeout(300);
+
+          const { width, height } = await getEditorDimensions(page, 0);
+
+          // The image is bigger than the page, so it has been scaled down to
+          // 75% of the page width.
+          expect(width).toEqual("75%");
+          expect(height).toEqual("auto");
+
+          const [bitmap] = await serializeBitmapDimensions(page);
+          expect(bitmap.width).toEqual(512);
+          expect(bitmap.height).toEqual(543);
+
+          await page.keyboard.down("Control");
+          await page.keyboard.press("a");
+          await page.keyboard.up("Control");
+          await page.waitForTimeout(10);
+
+          await page.keyboard.press("Backspace");
+        })
+      );
+    });
+
+    it("must load a SVG", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          if (browserName === "firefox") {
+            pending(
+              "Disabled in Firefox, because of https://bugzilla.mozilla.org/1553847."
+            );
+          }
+
+          const rect = await page.$eval(".annotationEditorLayer", el => {
+            // With Chrome something is wrong when serializing a DomRect,
+            // hence we extract the values and just return them.
+            const { x, y } = el.getBoundingClientRect();
+            return { x, y };
+          });
+
+          await page.mouse.click(rect.x + 100, rect.y + 100);
+          const input = await page.$("#stampEditorFileInput");
+          await input.uploadFile(
+            `${path.join(__dirname, "../images/firefox_logo.svg")}`
+          );
+
+          await page.waitForTimeout(300);
+
+          const { width, height } = await getEditorDimensions(page, 1);
+
+          expect(Math.round(parseFloat(width))).toEqual(40);
+          expect(height).toEqual("auto");
+
+          const [bitmap] = await serializeBitmapDimensions(page);
+          // The original size is 80x242 but to increase the resolution when it
+          // is rasterized we scale it up by 96 / 72
+          const ratio = 96 / 72; // PixelsPerInch.PDF_TO_CSS_UNITS
```

Comment:
Hard-coding this doesn't feel great, and it ought be available as follows.
```suggestion
          const ratio = window.pdfjsLib.PixelsPerInch.PDF_TO_CSS_UNITS;
```

Reply:
I looked for something like that and I didn't find it, thx.